crates/sui-rest-api/src/transactions/execution.rs

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::SocketAddr;
+use std::sync::Arc;
+
+use axum::extract::{Query, State};
+use sui_sdk2::types::framework::Coin;
+use sui_sdk2::types::{
+    Address, BalanceChange, CheckpointSequenceNumber, Object, Owner, SignedTransaction,
+    TransactionEffects, TransactionEvents, ValidatorAggregatedSignature,
+};
+use tap::Pipe;
+
+use crate::response::Bcs;
+use crate::Result;
+use crate::{accept::AcceptFormat, response::ResponseContent};
+
+pub const POST_EXECUTE_TRANSACTION_PATH: &str = "/transactions";
```

Comment:
should we name it `submit_transaction` or `excecute_transaction_block`? similar to https://docs.sui.io/references/sui-api/sui-graphql/reference/api/mutations/execute-transaction-block.
I assume eventually there will be a separate REST endpoint to query existing transactions, just to avoid confusion

Reply:
do you mean the route itself?

Generally in REST services you distinguish this via the HTTP method, eg POST vs GET. So I have a commit locally that adds txn querying off of the same route but using GET vs POST here.